src/instantsearch.js

```diff
@@ -1,119 +1,119 @@
+import AisPagination from './components/Pagination.vue';
+import AisPanel from './components/Panel.vue';
+import AisPoweredBy from './components/PoweredBy.vue';
+import AisRangeInput from './components/RangeInput.vue';
+import AisRatingMenu from './components/RatingMenu.vue';
+import AisRefinementList from './components/RefinementList.vue';
+import AisSearchState from './components/SearchState.vue';
+import AisSearchBox from './components/SearchBox.vue';
+import AisSnippet from './components/Snippet.vue';
+import AisSortBy from './components/SortBy.vue';
+import AisStats from './components/Stats.vue';
+import AisToggleRefinement from './components/ToggleRefinement.vue';
 
 const InstantSearch = {
   Component,
   // widgets:
-  Autocomplete,
-  Breadcrumb,
-  ClearRefinements,
-  Configure,
-  CurrentRefinements,
-  HierarchicalMenu,
-  Highlight,
-  Hits,
-  HitsPerPage,
-  Index,
-  InfiniteHits,
-  Menu,
-  MenuSelect,
-  NumericMenu,
-  Pagination,
-  Panel,
-  PoweredBy,
-  RangeInput,
-  RatingMenu,
-  RefinementList,
-  SearchState,
-  SearchBox,
-  Snippet,
-  SortBy,
-  Stats,
-  ToggleRefinement,
+  AisAutocomplete,
+  AisBreadcrumb,
+  AisClearRefinements,
+  AisConfigure,
+  AisCurrentRefinements,
+  AisHierarchicalMenu,
+  AisHighlight,
+  AisHits,
+  AisHitsPerPage,
+  AisInstantSearch,
+  AisInfiniteHits,
+  AisMenu,
+  AisMenuSelect,
+  AisNumericMenu,
+  AisPagination,
+  AisPanel,
+  AisPoweredBy,
+  AisRangeInput,
+  AisRatingMenu,
+  AisRefinementList,
+  AisSearchState,
+  AisSearchBox,
+  AisSnippet,
+  AisSortBy,
+  AisStats,
+  AisToggleRefinement,
 
   install(Vue) {
-    Vue.component('ais-autocomplete', Autocomplete);
-    Vue.component('ais-breadcrumb', Breadcrumb);
-    Vue.component('ais-clear-refinements', ClearRefinements);
-    Vue.component('ais-configure', Configure);
-    Vue.component('ais-current-refinements', CurrentRefinements);
-    Vue.component('ais-hierarchical-menu', HierarchicalMenu);
-    Vue.component('ais-highlight', Highlight);
-    Vue.component('ais-hits-per-page', HitsPerPage);
-    Vue.component('ais-hits', Hits);
-    Vue.component('ais-index', Index);
-    Vue.component('ais-infinite-hits', InfiniteHits);
-    Vue.component('ais-menu', Menu);
-    Vue.component('ais-menu-select', MenuSelect);
-    Vue.component('ais-numeric-menu', NumericMenu);
-    Vue.component('ais-pagination', Pagination);
-    Vue.component('ais-panel', Panel);
-    Vue.component('ais-powered-by', PoweredBy);
-    Vue.component('ais-range-input', RangeInput);
-    Vue.component('ais-rating-menu', RatingMenu);
-    Vue.component('ais-refinement-list', RefinementList);
-    Vue.component('ais-search-box', SearchBox);
-    Vue.component('ais-search-state', SearchState);
-    Vue.component('ais-snippet', Snippet);
-    Vue.component('ais-sort-by', SortBy);
-    Vue.component('ais-stats', Stats);
-    Vue.component('ais-toggle-refinement', ToggleRefinement);
+    Vue.component('ais-autocomplete', AisAutocomplete);
```

Comment:
Rather than name them here could we add the `name` property on each of them? To me it makes more sense to have them self-contained.

Reply:
Yes, I initially forgot that existed. Will change that usage to add the name on all of them!